src/main/scala/com/gu/mobile/notifications/football/GoalNotificationsService.scala

```diff
@@ -45,22 +46,13 @@ trait Rendering {
 
       <h2>Goal notifications history</h2>
       {
-        for (NotificationSent(when, notification, _) <- history collect { case n: NotificationSent => n }) yield
+        for (NotificationSent(when, notification) <- history collect { case n: NotificationSent => n }) yield
           <dl>
             <dt>Time sent</dt>
             <dd>{ when.toString() }</dd>
 
```

Comment:
The payloads and TTL are not available here anymore so i removed them.
I could extract most of the same information from the goalAlertPayload but I'm not sure if this is even being used. 
If I go this endpoint in both CODE and PROD I only see "Not contacted PA yet" so I guess this doesn't work?